.vscode/c_cpp_properties.json.default

```diff
@@ -0,0 +1,20 @@
+{
+    "configurations": [
+        {
+            "name": "stm8",
+            "includePath": [
+                "./src/controller",
+                "./src/common",
+                "./src/common/STM8S_StdPeriph_Lib/inc",
+                "C:\\Program Files\\SDCC\\include"
```

Comment:
Developer can rename c_cpp_properties.json.default to c_cpp_properties.json and modify includePath to SDCC\\include.